docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Carbon documentation
+
+This is the home for all process, contribution, and version-related documentation. These docs include information on project
+management, contribution, migration, reflections, and process insights.
+
+If you think something is missing, we encourage [contributions](#how-to-contribute).
+
+## Project management
+
+- [Endgame](/project-management/endgame.md)
+- [Versioning](/project-management/versioning.md)
+- [Support](/project-management/support.md)
+
+## How to contribute
+
+- [Developer handbook](/contributing/developer-handbook.md)
```

Comment:
Could you talk through some of the groupings? Not sure I understand all of them.

For example, Windows HCM is more related to support and definition and done over contributing. IBM Plex is more of a guide for using IBM Plex versus something folks do when working with the project